test/colony.js

```diff
@@ -1558,9 +1552,9 @@ contract("Colony", accounts => {
 
       // Run through the task flow
       await colony.submitTaskDeliverable(taskId, DELIVERABLE_HASH, { from: WORKER });
-      await colony.submitTaskWorkRating(taskId, WORKER_ROLE, RATING_2_SECRET, { from: EVALUATOR });
+      await colony.submitTaskWorkRating(taskId, WORKER_ROLE, RATING_2_SECRET, { from: MANAGER });
       await colony.submitTaskWorkRating(taskId, MANAGER_ROLE, RATING_1_SECRET, { from: WORKER });
-      await colony.revealTaskWorkRating(taskId, WORKER_ROLE, WORKER_RATING, RATING_2_SALT, { from: EVALUATOR });
+      await colony.revealTaskWorkRating(taskId, WORKER_ROLE, WORKER_RATING, RATING_2_SALT, { from: MANAGER });
```

Comment:
As above, let's keep the `EVALUATOR` constant reference here. It is whoever is assigned to the evaluator role that rates even if they happen to be the same person as the manager.

Reply:
Yup